Content.Server/Chat/SuicideSystem.cs

```diff
@@ -49,7 +49,7 @@ public override void Initialize()
     public bool Suicide(EntityUid victim)
     {
         // Can't suicide if we're already dead
-        if (!TryComp<MobStateComponent>(victim, out var mobState) || _mobState.IsDead(victim, mobState))
+        if (!TryComp<MobStateComponent>(victim, out var mobState) || _mobState.IsDead(victim, mobState) || _tagSystem.HasTag(victim, "CannotSuicideAny")) // Goobstation
```

Comment:
The tag parameter of HasTag forbids literal values
Use `private static readonly ProtoId<TagPrototype> CannotSuicideAny = "CannotSuicideAny";`
```suggestion
        if (!TryComp<MobStateComponent>(victim, out var mobState) || _mobState.IsDead(victim, mobState) || _tagSystem.HasTag(victim, CannotSuicideAny)) // Goobstation
```

Reply:
Fixed